harvard-lts/CURIOSity
# Arabic exhibit dashboard pages style updates

**Arabic exhibit dashboard pages style updates**
* * *

**JIRA Ticket**: https://github.com/harvard-lts/CURIOSity/issues/406

# What does this Pull Request do?
Tweaking a few styles for the dashboard pages when language is switched to Arabic

# How should this be tested?
Since we can't pull in "Coptica at Harvard" locally, this is a bit of a creative way to test out the changes:
* Log in to the "Coptica at Harvard" exhibit dashboard on QA and switch language to Arabic
* Inspect the code and add the new styles in a new style sheet in the "Style Editor" tab (since the styles are in sass, i converted them into css and attached the code at the bottom of this PR) *I also did this in Firefox
* Navigate through the various dashboard pages and make sure that nothing looks wonky; some examples to look for:
   * Sidebar menu items are flush right (was generating some additional padding on the right)
   * External link icon now has a little space between it and the word "Analytics"
   * Under General --> Basic Settings: there's now space between the checkbox and words "Published" and "Hide on Homepage" (closer to the bottom of the page)
   * Under Metadata: the "drag" icon is now on the right side (was previously appearing between the checkbox and field name)
   * Under Feature pages: subpages are represented with an indentation on the right side (it was still left-indented before like it is in the English version)
   * "Save changes" buttons are all on the left side (they were still sort of floating right)


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@VanessaVenti @enriquediaz 

# New CSS styles

```
 html[lang="ar"] #main-container ul.nav.sidenav.flex-column {
	 padding-right: 0;
}
 html[lang="ar"] #main-container ul.nav.sidenav.flex-column .hl__external-link::after {
	 margin-right: 5px;
}
 html[lang="ar"] .form-check-label {
	 margin-right: 20px;
}
 html[lang="ar"] .dd3-handle {
	 border-radius: 0 4px 4px 0;
	 right: 0;
}
 html[lang="ar"] .dd-item td:first-child .handle-wrap {
	 padding: 0.75rem 40px 0.75rem 0.75rem;
}
 html[lang="ar"] .dd-list .card-header, html[lang="ar"] .dd-item .card-header {
	 padding-right: 30px;
}
 html[lang="ar"] .form-check-inline {
	 margin-left: 0.75rem;
}
 html[lang="ar"] .float-right, html[lang="ar"] .primary-actions {
	 float: left !important;
}
 html[lang="ar"] .translation-edit-form .col-md-12 a.btn.mr-3 {
	 margin-right: 0 !important;
	 margin-left: 15px;
}
 html[lang="ar"] .translation-edit-form .translation-import-export-label {
	 padding-right: 0;
	 padding-left: 15px !important;
}
 html[lang="ar"] .translation-edit-form input#file.form-control {
	 border-radius: 0 4px 4px 0;
}
 html[lang="ar"] .translation-edit-form input.btn[value="Import translations"] {
	 border-radius: 4px 0 0 4px;
}
 @media (min-width: 481px) {
	 html[lang="ar"] .blacklight-exhibits-edit #import input#file {
		 border-radius: 0 4px 4px 0;
	}
	 html[lang="ar"] .blacklight-exhibits-edit #import .btn, html[lang="ar"] .blacklight-exhibits-edit #import .st-block-controls__button, html[lang="ar"] .blacklight-exhibits-edit #import .expanded-add-button .input-field input[type='submit'], html[lang="ar"] .expanded-add-button .input-field .blacklight-exhibits-edit #import input[type='submit'] {
		 border-radius: 4px 0 0 4px;
	}
}
```
